libtransmission/rpc-server.cc

```diff
@@ -51,6 +53,12 @@
 #include "libtransmission/variant.h"
 #include "libtransmission/web-utils.h"
 
+#ifdef WITH_OPENSSL
```

Comment:
Here and elsewhere: the macro is defined as `WITH_LIBEVENT_OPENSSL` in CMakeLists.txt.